doc/SpectatorView.Setup.md

```diff
@@ -48,6 +48,14 @@ Not all spatial alignment strategies support all platforms. See the chart below
 3. In the Android, iOS and WSA unity player settings, add the **SPATIALALIGNMENT_ASA** preprocessor directive. (This is located via Build Settings -> Player Settings -> Other Settings -> 'Scripting Defined Symbols')
 4. When in Unity preparing to build your application, set the Account Domain, Account Id and Account Key for the Spatial Anchors Localizer using values you obtained creating an azure spatial anchors account above. These values are set in the SpatialAnchorsCoordinateLocalizationInitializer MonoBehaviour defined, which will exist for the Android and iOS experiences.
 
+#### Azure Spatial Anchors on iOS
+If you are building Azure Spatial Anchors on iOS, you will need to take some additional steps after generating your XCode project through Unity. After exporting an iOS version of your application in Unity, do the following:
+
+1. In the terminal, navigate to your xcode project folder.
+2. Run **'pod install --repo-update'** in the terminal when in your xcode project folder.

```

Comment:
Should we create a task to automate this in the future? This command could be run from a Unity post-build script.

Reply:
Possibly, really this is ASA, and I think they are working on it.

Reply:
Going to check this in for now, based on user feedback we can decide what to do here.